package.json

```diff
@@ -15,10 +15,10 @@
     "cordova:platform"
   ],
   "scripts": {
-    "test": "npm run eslint && npm run jasmine",
-    "jasmine": "npm run objc-tests && npm run jasmine-tests",
-    "objc-tests": "jasmine tests/spec/cordovalib.spec.js",
-    "jasmine-tests": "jasmine tests/spec/create.spec.js tests/spec/platform.spec.js",
+    "test": "npm run eslint && npm run test:coverage",
+    "test:objc": "jasmine  --config=tests/spec/objc.json",
+    "test:component": "jasmine --config=tests/spec/component.json",
+    "test:coverage": "nyc jasmine --config=tests/spec/coverage.json",
```

Comment:
naming the file `all.json` would have been more obvious to me. But that's only nitpicking.